extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,6 +150,31 @@ export class ConsentConfig {
       }
     }
 
+    userAssert(
```

Comment:
this assert might be not that useful. we can either have fallbacks, or fail individually.

Reply:
removed